scripts/templates/APKBUILD.template

```diff
@@ -16,25 +16,25 @@ subpackages="$pkgname-dev $pkgname-doc"
 source="msquic-$pkgver.tar.gz::ARCHIVE_URI_PLACEHOLDER
 	clog-$_clog.tar.gz::https://github.com/microsoft/CLOG/archive/$_clog.tar.gz
 	gtest-$_gtest.tar.gz::https://github.com/google/googletest/archive/$_gtest.tar.gz
-	openssl3-$_openssl3.tar.gz::https://github.com/quictls/openssl/archive/$_openssl3.tar.gz
+	quictls3-$_quictls3.tar.gz::https://github.com/quictls/openssl/archive/$_quictls3.tar.gz
 	"
 builddir="$srcdir/msquic-SHA_PLACEHOLDER"
 
 prepare() {
 	default_prepare
 
 	cd "$builddir/submodules"
-	rm -rf clog googletest openssl openssl3 xdp-for-windows
+	rm -rf clog googletest openssl quictls3 xdp-for-windows
```

Comment:
Why is this keeping around openssl? Shouldn't it be "quictls"?

Reply:
for the same reason as above, this also needs to stay put :(